recipes/jsonnet/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+
+class JsonnetConan(ConanFile):
+    name = "jsonnet"
+    version = "0.17.0"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/jsonnet"
+    description = "Jsonnet - The data templating language"
+    topics = ("conan", "jsonnet")
```

Comment:
```suggestion
    topics = ("config", "json", "functional", "configuration")
```

From jsonnet project